contrib/util/language_translations/currentConstants.txt

```diff
@@ -9180,7 +9180,7 @@ SMS OK
 SMS send before
 SMS Sent
 SMS Text Usable Tags
-SMS Text, Usable Tag: ***NAME***, ***PROVIDER***, ***DATE***, ***STARTTIME***, ***ENDTIME***<br> i.e. Dear ***NAME***
+SMS Text, Usable Tag: ***NAME***, ***PROVIDER***, ***DATE***, ***STARTTIME***, ***ENDTIME***<br /> i.e. Dear ***NAME***
 SMS Text, Usable Tags: 
```

Comment:
@tywrenn , You will need to revert all the changes (just 3) in this currentConstants.txt script. This script is created via a translation pipeline and are never touched directly (note it keeps old constants also to support backward compatibility).

Reply:
@tywrenn, also same for the currentLanguage_utf8.sql and current_spreadsheet.tsv files. Just recommend in copying back the files from current master.